src/tests/ftest/util/server_utils_params.py

```diff
@@ -5,7 +5,8 @@
 """
 import os
 
-from command_utils_base import BasicParameter, LogParameter, TransportCredentials, YamlParameters
+from command_utils_base import (BasicParameter, LogParameter,
+                                TransportCredentials, YamlParameters)
```

Comment:
You might have a different version of isort if the githooks are changing this.

You can check with
```
python3 -m isort --version
```

And the current you should have is 5.13.2, from `utils/cq/requirements.txt`